configs/det/dbnet/db++_r50_icdar15.yaml

```diff
@@ -84,9 +84,12 @@ train:
       - RandomColorAdjust:
           brightness: 0.1255  # 32.0 / 255
           saturation: 0.5
-      - IaaAugment:
-          Fliplr: { p: 0.5 }
-          Affine: { rotate: [ -10, 10 ], p: 1.0 }
+      - RandomHorizontalFlip:
+          p: 0.5
+      - RandomRotate:
+          degrees: [ -10, 10 ]
+          expand_canvas: False
+          p: 1.0
```

Comment:
Is the time cost of RandomHorizontalFlip+RandomRotate the same as that of IaaAugment? How is its impact on the Throughout.

Reply:
Because `IaaAugment`'s implementation is roughly the same and based on `OpenCV`, the performance of our version should be the same of even better (`imgaug` has a lot of overhead in checkings). The motivation of this is to get rid of long dead `imgaug` library.

Reply:
I see.  It would be better to reassure by testing. Could you please also post the test results (hmean and fps after training with a few epochs like 300) that compare the new Random Operations with the previous IaaAugment?

Reply:
The first thing I did is made sure that new augmentations will not affect our models' generalization abilities:
https://github.com/mindspore-lab/mindocr/pull/292#issuecomment-1554344790

Also, I didn't notice any decrease in speed during training. I can speed benchmark these operations independently to make sure they are on par with `imgaug`.

Reply:
Completed the time and throughput benchmarking for the two operations on 300 x 300, 600 x 600, and 1000 x 1000 images. 1000 iterations were performed for each image size. Total Time and Throughput are mentioned below:
|              **Method**             |              **300 x 300**              |              **600 x 600**             |             **1000 x 1000**            |
|:-----------------------------------:|:---------------------------------------:|:--------------------------------------:|:--------------------------------------:|
| IaaAugment (imgaug)                 | 1.908 seconds<br>524.073 imgs/second  | 3.024 seconds<br>330.721 imgs/second | 5.821 seconds<br>171.795 mgs/second |
| **RandomRotate & RandomHorizontalFlip** | 0.523 seconds<br>1913.504 imgs/second | 1.232 seconds<br>811.757 imgs/second | 3.035 seconds<br>329.456 imgs/second |